packages/cli/src/commands/generate/dbAuth/dbAuth.js

```diff
@@ -126,6 +148,12 @@ export const files = ({
         webPathSection: 'pages',
         generator: 'dbAuth',
         templatePath: 'forgotPassword.tsx.template',
+        templateVars: {
+          usernameLowercase,
+          usernameCapitalised,
+          passwordLowercase,
+          passwordCapitalised,
+        },
```

Comment:
I'd do this earlier in the file:

```
const templateVars = {
  usernameLowercase,
  usernameCapitalised,
  passwordLowercase,
  passwordCapitalised,
},
```

And then this will just become:

```suggestion
        templateVars,
```